ckb-bench/src/bench.rs

```diff
@@ -10,27 +10,51 @@ use ckb_testkit::{Node, User};
 use crossbeam_channel::{Receiver, Sender};
 use lru::LruCache;
 use std::collections::HashMap;
-use std::time::Instant;
+use std::sync::{Arc};
+use std::time::{Duration, Instant};
+use tokio::sync::Semaphore;
+use futures::stream::FuturesUnordered;
+use futures::{FutureExt, StreamExt};
+use tokio::time::sleep as async_sleep;
+use crate::utils::maybe_retry_send_transaction_async;
+use std::sync::atomic::{AtomicUsize, Ordering};
 
 pub struct LiveCellProducer {
     users: Vec<User>,
     nodes: Vec<Node>,
     seen_out_points: LruCache<OutPoint, Instant>,
 }
 
-// TODO Add more logs
 impl LiveCellProducer {
     pub fn new(users: Vec<User>, nodes: Vec<Node>) -> Self {
         let n_users = users.len();
+
+        let mut max_count = 1;
+        for i in (0..=users.len()-1).step_by(20) {
+            let cell_count = users.get(i).expect("out of bound").get_spendable_single_secp256k1_cells(&nodes[0]).len();
+            if cell_count > max_count && cell_count <= 10000 {
+                max_count = cell_count;
+            }
+            ckb_testkit::debug!("idx:{}:cell_size:{}",i,cell_count)
+        }
+        ckb_testkit::debug!("user max count cell:{}",max_count);
```

Comment:
Why `step_by(20)`? Why don't update `max_count` when `cell_count <= 10000`?

Reply:
The `cell_count` here represents the number of unused cells cached for each user, and the total cache size is: `cell_count` * `user_size`. 
This is because the previous stress test code caused some users to have 0 cells and some users to have a large number of cells after the stress test was completed. Therefore, the `step_by: 20` is used here to randomly select some cells and take the maximum quantity to represent `cell_count`. 
The `cell_count` is less than or equal to 10,000 to prevent some cells from having too many cells, causing the cell_count * user_size to be too large. 

So I might make the following adjustments:
1. The stress test code may need to be adjusted here to match the inputs and outputs of the test one by one.
2. Optimize this part of the code, replacing `cell_count` with `user_unused_cell_count_cache`.
3. Add comments to the code.